packages/plugins/rubygem-foreman_openscap/rubygem-foreman_openscap.spec

```diff
@@ -108,6 +108,12 @@ cp -pa .%{gem_dir}/* \
 exit 0
 
 %changelog
+* Wed Jun 26 2019 vagrant 1.0.3-1
```

Comment:
Could you correct the name here? It may be fine to squash it with the 1.0.2 update